Big-Spender/readme.md

```diff
@@ -125,6 +132,8 @@ CREATE YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERIES HERE
+INSERT INTO spends (date, transaction_number, supplier_invoice_number, supplier, expense_type, expense_area, description, amount)
+VALUES ('2021-08-19', 38104091, 3780119655, 'Dell', 'Hardware', 'IT', 'Computer Hardware Dell', 32000);
```

Comment:
so how would you insert into spends with Hardware or IT, which is not in the corresponding tables already? What would you need to do in order insert these into the corresponding tables. You might want to go back and give this one a try :)